pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1440,6 +1440,22 @@ type GatewaySpec struct {
 	HostNetwork *bool `json:"hostNetwork,omitempty"`
 }
 
+// EndpointAddress is a tuple that describes a single IP address or host name. This is a subset of
+// Kubernetes's v1.EndpointAddress.
+// +structType=atomic
+type EndpointAddress struct {
+	// The IP of this endpoint.
+	// May not be loopback (127.0.0.0/8), link-local (169.254.0.0/16),
```

Comment:
Are all these comments relevant? Seems like we shouldn't really care what type of address is given as long as it is reachable.